lesson1/src/arrays/arrayProblems.js

```diff
@@ -22,25 +25,52 @@ class ArrayProblems {
    */
   static getMax(array) {
     // todo: do stuff here
+    return array.reduce((acc, curr) => {
+      return acc > curr ? acc : curr;
+    })
   }
 
   /*
    Can you implement getMax again without using any js collection method?
    */
   static getMax2(array) {
     // todo: 🙌 do magic !
+    let result;
+    let i = 0;
+    let j = 0;
+    while(i < array.length){
+      while(j < array.length -1){
+        if(array[i] > array[j]){
+          result = array[i];
+        }else if(array[j] > array[i]){
+          result = array[j];
+        }
+        j++
+      }
+      i++
+      j = i;
+    }
+    // for (let i = 0; i < array.length; i++) {
+    //   for (let j = i; j < array.length - 1; j++) {
+    //     // todo: do stuff here
+    //     if (array[j] > array[i]) {
```

Comment:
it works but goo to think about what you're doing that might reduce the amount of effort and make it easier to the computer(to understand me haha), thanks